README.md

```diff
@@ -207,9 +207,15 @@ Looking for conceptual tutorials and explanations? We have some additional artic
 
 &nbsp;
 
-## Pre-training
+## Pretraining
 
-Porting from Lit-LLaMA in progress 👷
+
+
+We provide [a simple training script based on Fabric](pretrain/redpajama.py) if you want to venture into pretraining on RedPajama, a reproduction of the original LLaMA dataset. Conversion scripts for our optimized streaming `PackedDataset` are included.
+
+Follow this guide to start pretraining on the RedPajama dataset:
+
+- [Pretrain on RedPajama](tutorials/pretrain_redpajama.md)
```

Comment:
Leaving a comment here in case you'd like a reminder for adding the openwebtext reference here from your other PR